services/thumbnails/pkg/thumbnail/thumbnail.go

```diff
@@ -2,6 +2,7 @@ package thumbnail
 
 import (
 	"bytes"
+	errors2 "github.com/owncloud/ocis/v2/services/thumbnails/pkg/errors"
```

Comment:
errors2 is not so nice, could we use something like `thumbnailerErrors`?

Reply:
autogenerated by ide :shrug: ...